angular-apollo-tailwind/src/app/pull-requests/components/pull-requests-list/pull-requests-list.component.ts

```diff
@@ -11,6 +11,10 @@ export class PullRequestsListComponent {
 
   readonly baseDate = new Date();
 
+  toDate(date: any): Date {
```

Comment:
Could we use a [DatePipe](https://angular.io/api/common/DatePipe) instead? 

That way it only runs when it detects a change in the input

Reply:
Okay let me work on a workaround